docs/importing.md

```diff
@@ -0,0 +1,68 @@
+# Importing existing Data into Chronicle
+
+## Importing Data into Chronicle
+
+Chronicle provides a CLI command `import` to load data from a JSON-LD file that
+contains an array of Chronicle Operations. These operations can be used to
+create agents, namespaces, and other resources in the Chronicle database. For
+more information about the data format of the import file, see [Recording Provenance](../recording_provenance/#importing-data-into-chronicle).
+
+### Local development environment
+
+To import data into Chronicle, follow these steps:
+
+- Create a JSON-LD file with an array of Chronicle Operations.
+- Save the file in the Chronicle root directory with a unique name, for example,
+  `import.json`.
+- Run the following command:
+
+  ```bash
+  chronicle --bin chronicle -- import <namespace-id> <namespace-uuid> <json-file-url>
+  ```
+
+If the data is successfully imported, the Chronicle Operations will be added
+to the Chronicle database.
+
+### Docker compose environment
+
+To import data into Chronicle, follow these steps:
+
+- Create a JSON-LD file with an array of Chronicle Operations.
+- Add a bind mount to your docker compose file mounting your import file in the
+  chronicle container, if you're using the chronicle-examples or
+  chronicle-bootstrap configuration then this will for the
+  `chronicle-sawtooth-api` container, defined in  `docker/chronicle.yaml`.
```

Comment:
If they're running `make run-my-domain` without `stl` in there then it'll be `docker/chronicle-domain.yaml` that they need to edit instead.